packages/api/lambdas/sf-semaphore-down.js

```diff
@@ -44,11 +46,12 @@ const getEventMessage = (event) => JSON.parse(get(event, 'detail.output', '{}'))
  *   - Workflow is not in a terminal state
  *
  * @param {Object} event - A workflow execution event
+ * @param {Object} eventMessage - A cumulus event message
  * @returns {boolean} True if workflow execution semaphore should be decremented
  */
-const isDecrementEvent = (event) =>
+const isDecrementEvent = (event, eventMessage) =>
   isSfExecutionEvent(event)
-  && hasQueueAndExecutionLimit(getEventMessage(event))
+  && hasQueueAndExecutionLimit(eventMessage)
```

Comment:
Not sure if this matters, but maybe we could flip the order of this and the next condition so that it checks everything in the `event` is valid before checking the execution message?

Reply:
Sure, that's a reasonable change.

Reply:
380a681